common/app/views/fragments/analytics/google.scala.html

```diff
@@ -5,6 +5,20 @@
 
 (function() {
 
+    @*
```

Comment:
Luckly this code is already excuted earlier in the javascript loading cycle in https://github.com/guardian/frontend/blob/master/common/app/templates/inlineJS/nonBlocking/getUserData.scala.js and should be available in `window.guardian.config.user` if they are logged in!